### ISSUE TITLE
pranges selection

### ISSUE THREAD

**Issue by lgatto** [ComputationalProteomicsUnit/Pbase#40]
Now that we will support multiple `pranges` (see #38), we need a way to way to define which one to use. 
Suggestion is to have a optional  `character(1)`  `pcol` argument where necessary. If missing, then the name of the first `pranges` is used. If there is no `pranges` or `pcol` is not among `pvarLabels`, then an error is thrown.

Similarly, when a new `pranges` is created, we need to consider the fact that a range with the same name already exists. My suggestion is to warn the user and return the unchanged object, rather than overwriting silently. Rather than having an argument that defines the name of the newly created, I would suggest we make sure it easy to replace `pranges` names with a replacement method. 
```r
names(pranges(p)) <- c("newNames", ...)
```

**Comment by jotsetung** [comment]
Makes sense. Means the `mapToGenome` will have an additional `pcol` argument. I'll implement that for `mapToGenome,Proteins,EnsDb`.

**Comment by jotsetung** [comment]
`pcol` parameter added to `mapToGenome` and `pmapToGenome` (https://github.com/ComputationalProteomicsUnit/Pbase/commit/c0a43857be3562b7e41e4c0da957896efdd4885c).

**Comment by jotsetung** [comment]
I've added a `.checkPcol` utility function to check for the `pcol` parameter: throws an error if provided `pcol` does not match any `pvarLabels` or if no protein ranges are present. If `pcol` is missing it returns `pvarLabels(x)[1]` as suggested above.
